includes/class-sensei-admin.php

```diff
@@ -295,14 +296,34 @@ public function admin_styles_global( $hook ) {
 		Sensei()->assets->register( 'jquery-modal', '../vendor/jquery-modal-0.9.1/jquery.modal.min.css' );
 
 		// Test for Write Panel Pages
-		if ( ( ( isset( $post_type ) && in_array( $post_type, $allowed_post_types ) ) && ( isset( $hook ) && in_array( $hook, $allowed_post_type_pages ) ) ) || ( isset( $_GET['page'] ) && in_array( $_GET['page'], $allowed_pages ) ) ) {
-
+		if ( $this->are_custom_admin_styles_allowed( $post_type, $hook, get_current_screen() ) ) {
 			Sensei()->assets->enqueue( 'sensei-admin-custom', 'css/admin-custom.css', [], 'screen' );
-
 		}
 
 	}
 
+	/**
+	 * Check if it is allowed to enqueue admin custom styles.
+	 *
+	 * @param string         $post_type The post type slug.
+	 * @param string         $hook_suffix The current admin page.
+	 * @param WP_Screen|null $screen The current screen.
+	 * @return bool Returns true if admin custom styles are allowed.
+	 */
+	private function are_custom_admin_styles_allowed( $post_type, $hook_suffix, $screen ) {
+		$allowed_post_types      = apply_filters( 'sensei_scripts_allowed_post_types', array( 'lesson', 'course', 'question' ) );
+		$allowed_post_type_pages = apply_filters( 'sensei_scripts_allowed_post_type_pages', array( 'edit.php', 'post-new.php', 'post.php', 'edit-tags.php' ) );
+		$allowed_pages           = apply_filters( 'sensei_scripts_allowed_pages', array( 'sensei_grading', 'sensei_analysis', 'sensei_learners', 'sensei_updates', 'sensei-settings', $this->lesson_order_page_slug, $this->course_order_page_slug ) );
```

Comment:
Nice! I like that you've decided to extract this into a method. Just a reminder to clean up the filters from lines 283-285.